types/stripe-js/elements/fpx-bank.d.ts

```diff
@@ -0,0 +1,73 @@
+///<reference path='./base.d.ts' />
+
+declare module '@stripe/stripe-js' {
+  type StripeFpxBankElement = StripeElementBase & {
+    /**
+     * The change event is triggered when the `Element`'s value changes.
+     */
+    on(
+      eventType: 'change',
+      handler: (event: StripeFpxBankElementChangeEvent) => any
+    ): void;
```

Comment:
Should this be
```suggestion
    ): StripeFpxBankElement;
```
? I think so, it looks like the other element types have this wrong too.

Reply:
Updating them all.